### DISCUSSION TITLE
what are the steps for Germline and Somatic CNV detection using CNVkit?

### DISCUSSION THREAD

**Discussion by ahmadsam66** [etal/cnvkit#1019] [General]
Hello,

I have some basic questions regarding the CNV calling using CNVkit. I have tried to find the CNV caused between normal and tumor samples, but it was not clear which steps should be done respectively.

1- For WGS (Somatic) tumor and normal samples, Should I first run the CNVkit batch then CNVkit call ? or "batch" is enough?

2- What would be the WGS Germline CNV calling steps?

3- How can I find filter the founded CNV, I mean the criteria for cnr or cnn ? ( I mean the significant CNV for the genes).

Thanks a lot.

**Comment by etal** [comment]
`cnvkit.py batch -m wgs` should be enough to get useful calls. Then you can use the other commands individually to adjust details or fit to processing to your workflow. If you want details about specific genes in your output, I recommend the `genemetrics` command.
